packages/trader/src/App/Components/Elements/market-countdown-timer.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { Localize } from '@deriv/translations';
+import { WS } from 'Services/ws-methods';
+import { connect } from 'Stores/connect';
+
+// check market in coming 7 days
+const days_to_check_before_exit = 7;
+
+const getTradingTimes = async target_time => {
+    return new Promise(resolve => {
+        WS.tradingTimes(target_time).then(data => {
+            if (data.error) resolve({ api_initial_load_error: data.error.message });
+            resolve(data);
+        });
+    });
+};
+
+const getSymbol = (target_symbol, trading_times) => {
+    let symbol;
+    const { markets } = trading_times;
+    for (let i = 0; i < markets.length; i++) {
+        const { submarkets } = markets[i];
+        for (let j = 0; j < submarkets.length; j++) {
+            const { symbols } = submarkets[j];
+            symbol = symbols.find(item => item.symbol === target_symbol);
```

Comment:
Can we declare `symbol` as local variable and return it if found?

Reply:
No because it violates this eslint rule: `consistent-return`

Reply:
That's okay. We can return `undefined` at the end of the function.
Having this line `if (symbol !== undefined) break;` looks bit confusing IMO.